src/components/Chip.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import css from '@styled-system/css';
+import { Box as BoxPrimitive, BoxProps as BoxPrimitiveProps } from 'mdlz-prmtz';
+import { variant, Prop } from '@modulz/radix-system';
+import { setLightness } from 'polished';
+import {theme} from '../theme';
+
+type ChipProps = BoxPrimitiveProps & {
+  variant?: Prop<'gray' | 'primary' | 'blue' | 'green' | 'purple' | 'orange'>;
+  size?: Prop<0 | 1 | 2>;
+  truncate?: Prop<boolean>;
+};
+
+export const Chip = React.forwardRef<HTMLDivElement, ChipProps>((props, ref) => (
+  <BoxPrimitive
+    {...props}
+    ref={ref}
+    css={[
+      css({
+        fontWeight: 600,
+        margin: 1,
+        alignSelf: 'flex-start',
+        display: 'block',
+        minWidth: 4,
+        textAlign: 'center',
+        borderRadius: 5,
+      }),
+      variant({
+        variant: {
+          gray: {
+            color: 'grays.7',
+            backgroundColor: setLightness(0.90, theme.colors.gray),
+          },
+          primary: {
+            color: 'primary',
+            backgroundColor: setLightness(0.90, theme.colors.primary),
+          },
+          blue: {
+            color: 'blue',
+            backgroundColor: setLightness(0.90, theme.colors.blue),
+          },
+          green: {
+            color: 'green',
+            backgroundColor: setLightness(0.90, theme.colors.green),
+          },
+          purple: {
+            color: 'purple',
+            backgroundColor: setLightness(0.90, theme.colors.purple),
+          },
+          orange: {
+            color: 'orange',
+            backgroundColor: setLightness(0.90, theme.colors.orange),
+          }
+        },
+      }),
+      variant({
+        size: {
+          0: {
+            fontSize: 0,
```

Comment:
Checking Zeplin the based font-size is 14px (fontSize: 1) instead of  12px (fontSize: 0)